store/gaskv/store.go

```diff
@@ -6,63 +6,95 @@ import (
 	"cosmossdk.io/store/types"
 )
 
+// ObjectValueLength is the emulated number of bytes for storing transient objects in gas accounting.
+const ObjectValueLength = 16
```

Comment:
i feel like this value deserves more justification. seems kinda arbitrary. is it worth exploring using a number more related to the actual object size?